bigbone/src/main/kotlin/social/bigbone/api/entity/PreviewCard.kt

```diff
@@ -87,22 +86,16 @@ data class PreviewCard(
     /**
      * Used for photo embeds, instead of custom html.
      */
-    @SerialName("method_url")
-    val methodUrl: String = "",
+    @SerialName("embed_url")
+    val embedUrl: String = "",
 
     /**
      * A hash computed by the BlurHash algorithm,
      * for generating colorful preview thumbnails
      * when media has not been downloaded yet.
      */
     @SerialName("blurhash")
-    val blurhash: String? = null,
-
-    /**
-     * Usage statistics for given days (typically the past week).
-     */
-    @SerialName("history")
-    val history: List<History> = emptyList()
```

Comment:
I can see the `history` property as part of the [`Trends::Link`](https://docs.joinmastodon.org/entities/PreviewCard/#trends-link) payload but don’t quite get what that does and where to expect it.
Can you explain that to me?

Reply:
I think the Mastodon documentation is a bit misleading here. Basically it is like this:
- `PreviewCard` is used in the Status API
- `Trend::Link` is used in the Trends API
- Both entities are almost equal, they differ in the `history` attribute only (which is used in `Trend::Link`

Does that help?

Reply:
Ah, yes, it does! Thank you 👍

Reply:
For the Trends API I am currently implementing as part of https://github.com/andregasser/bigbone/issues/297, I'll probably implement a separate `TrendLink` entity, to allow it to evolve separately.